Makefile

```diff
@@ -315,7 +317,12 @@ test: unit-test e2e-test
 .PHONY: unit-test
 unit-test: envtest
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(TEST_SRC) -v  -coverprofile cover.out
+CLEANFILES += cover.out
```

Comment:
in reality i am not sure we need the output `-coverprofile cover.out` from unit-test
since it is already "-v" should have seen all from stdout

Reply:
Probably, but this is unrelated to the change.